src/core/utils/helpers/general.ts

```diff
@@ -102,7 +102,7 @@ export const usePrevious = <Type>(value: Type) => {
 };
 
 export const convertPostIdToFaustId = (postId: Pid) => {
-  const matches = postId.match(/^[0-9]+-[a-z]+:([0-9]+)$/);
+  const matches = postId.match(/^[0-9]+-[a-z]+:([a-zA-Z0-9]+)$/);
```

Comment:
Consider adding a TODO here explaining that there is thoughtwork here in progress.

Reply:
👍